v5_ws_public.go

```diff
@@ -148,7 +148,6 @@ func (s *V5WebsocketPublicService) Start(ctx context.Context, errHandler ErrHand
 		defer s.connection.Close()
 
 		_ = s.connection.SetReadDeadline(time.Now().Add(60 * time.Second))
-		_ = s.connection.SetWriteDeadline(time.Now().Add(60 * time.Second))
```

Comment:
📝 This is unnecessary because s.writeMessage is called for each ping and s.connection.SetWriteDeadline is called.